step.yml

```diff
@@ -67,19 +67,26 @@ inputs:
         - "yes"
         - "no"
       is_required: true
-  - additional_params: 
+  - additional_params:
     opts:
       title: "Additional parameters"
       summary: "The flags from this input field will be appended to the `flutter test` command."
       description: "The flags from this input field will be appended to the `flutter test` command."
+  - tests_path_pattern:
+    opts:
+      title: "Test files pattern"
+      summary: "This pattern will be used as the last argument appended to the `flutter test` command."
```

Comment:
This pattern is used as the last argument appended to the `flutter test` command.